middleware/admin/verify_attestation.py

```diff
@@ -138,6 +160,7 @@ def do_verify_attestation(options):
                                   mh_len + UD_VALUE_LENGTH + PUBKEY_COMPRESSED_LENGTH +
                                   SIGNER_HASH_LENGTH + SIGNER_ITERATION_LENGTH]
     signer_iteration = int.from_bytes(signer_iteration, byteorder='big', signed=False)
+    ui_version = ui_message[mh_len - 3:mh_len]
```

Comment:
If we end up using regular expressions, then we can use a group to gather the version from the header directly without having to use magic numbers, e.g., `re.match(b"^HSM:UI:(5.[0-9])$", "HSM:UI:5.3").group(1)`.